book.cmd/src/main/java/com/happystays/book/cmd/api/commands/BookCommand.java

```diff
@@ -0,0 +1,21 @@
+package com.happystays.book.cmd.api.commands;
+
+import com.happystay.cqrs.core.commands.BaseCommand;
+import lombok.Data;
+
+@Data
+public class BookCommand extends BaseCommand {
+    private int numberOfRooms;
+    private String ratePlanId;
+    private int guestcount;
+    private int guestAge;
+    private String startDate;
+    private String endDate;
+    private PaymentInfo paymentInfo;
+    private String hotelCode;
+    private CustomerInfo customerInfo;
+    private CancelInfo cancelInfo;
+    private String guaranteeIndicator;
```

Comment:
what is this guaranteeIndicator? we wont get this UI. I this we may need to hardcode this value in DB as we wont get this either from UI or in Book RS

Reply:
Will discuss and remove if not needed

Reply:
Removed the field